shtdlib.sh

```diff
@@ -815,7 +850,7 @@ function copy_file {
         debug 10 "Found globbing pattern in ${1}"
         # Make sure permissions and owner are OK
         set_file_perm_owner "${source}" "${owner_group}" "${perm}"
-        if ${force_overwrite} ; then
+        if ${force_overwrite:-false} ; then
```

Comment:
Quotes? Maybe?

Reply:
I don't think so, since true/false are actual commands and if uses their return status (0 or 1) adding quotes would make if try to interpret the string right ?

Reply:
I had to test to be sure, but looks like it will still work with or without quotes.  Up to you.

Reply:
let me see if it works with true/false correctly with the quotes, if it does then I'll change